piff/stats.py

```diff
@@ -689,9 +689,8 @@ def compute(self, psf, stars, logger=None):
         # Also write any other properties saved in the stars.
         self.props = {}
         prop_keys = list(stars[0].data.properties)
-        # Already did the position ones.
-        for key in [ 'x', 'y', 'u', 'v' ]:
-            prop_keys.remove(key)
+        # Remove all the position ones, which are handled separately.
+        prop_keys = [key for key in prop_keys if key not in ['x', 'y', 'u', 'v', 'ra', 'dec']]
```

Comment:
Just curious: why is this way better (besides including 'ra' and 'dec')?

Reply:
Because ra, dec aren't necessarily in there.  And python doesn't have a very easy "remove if present" for lists like it does for dicts and sets (`d.pop(key, None)`)